src/components/BlockQuote/BlockQuoteParagraph.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { css } from 'glamor'
+
+import { mUp } from '../../theme/mediaQueries'
+import { fontFamilies } from '../../theme/fonts'
+
+const styles = {
+  quote: css({
+    backgroundColor: '#f7f7f7',
+    margin: 0,
+    padding: '0 15px 12px 15px',
+    fontSize: '15px',
```

Comment:
Replace fontSize with
`...sansSerifRegular15,`

(Not a real improvement for your use-case now, but that'll help to consolidate fontSize/lineHeight combos at some point).